fabcar/startFabric.sh

```diff
@@ -109,8 +109,25 @@ Go:
   Start by changing into the "go" directory:
     cd go
 
+  Next, set hosts:
+    sudo vim /etc/hosts
+
+  add Under the content into  /etc/hosts
+    ip	peer0.org1.example.com  
+    ip	peer0.org2.example.com  
+    ip	peer0.org2.example.com  
+
+    for example:
+    127.0.0.1       peer0.org1.example.com  peer0.org2.example.com  orderer.example.com
```

Comment:
what is the need to add this entry to /etc/hosts ?

Last time, I tried , `go run fabcar.go` works without this. ( still need `export DISCOVERY_AS_LOCALHOST=true`)

Reply:
the need for this in hosts entry , is still not clear to me. 

`127.0.0.1       peer0.org1.example.com  peer0.org2.example.com  orderer.example.com`